tensorflow_lite_support/ios/task/vision/sources/TFLImageClassifier.m

```diff
@@ -100,6 +100,12 @@ - (nullable TFLClassificationResult *)classifyWithGMLImage:(GMLImage *)image
 - (nullable TFLClassificationResult *)classifyWithGMLImage:(GMLImage *)image
                                           regionOfInterest:(CGRect)roi
                                                      error:(NSError *_Nullable *)error {
+  if (image == nil) {
+    if (error != nil) {
+      *error = [TFLCommonUtils customErrorWithCode:TFLSupportErrorCodeInvalidArgumentError
+                                       description:@"GMLImage argument cannot be nil."];
```

Comment:
@khanhlvg Long back we had a discussion that the error null check here should be incorporated into `[TFLCommonUtils customErrorWithCode:]` in order to remove redundant error check every time we have to create an error. I had made this change but seems to have been reversed in a later commit while some PR was merged. I am not sure if this was accidental. If it was, can I incorporate the null error check into the method in a subsequent PR?

Reply:
Yes please incorporate the null check into TFLCommonUtils.

One thing that can cause the reverse is that we import the PR to merge internally, not on GitHub. Therefore, if you add a commit to a PR after we initiated the PR import process, that commit will end up not being included in the merge.

Reply:
Okay. I will

Reply:
Please let me know once you've added this refactor. The rest of the code LGTM so once that's done, I'll import this PR for internal review.

Reply:
This will bloat up the PR and change a lot of files other than image classifier since the method calls are in a lot of places. But they are minor changes. Is that okay?

Reply:
Thought I would do it in a follow up PR.

Reply:
I have incorporated the error refractor. LMK if it's a lot of code.